### DISCUSSION TITLE
[BP#78] Compression des SVG ?

### DISCUSSION THREAD

**Discussion by Julien-Elie** [cnumr/best-practices#544] [Bonnes pratiques]
Le titre de la BP mentionne les images SVG mais ni la description ni l'exemple n'en font mention par la suite.

**Comment by ldevernay** [comment]
En complément, c'est bien de mettre l'accent sur le brotli mais ce serait intéressant de mentionner les autres : https://paulcalvano.com/2024-03-19-choosing-between-gzip-brotli-and-zstandard-compression/
